examples/tutorials/hybrid_demucs_tutorial.py

```diff
@@ -0,0 +1,360 @@
+    figure.suptitle(title)
+    plt.colorbar(img, ax=axis)
+    plt.show()
+
+
+######################################################################
+# 5. Run Model
+# ------------
+# 
+# Finally, we run the model and store the separate source files in a
+# directory
+# 
+# As a test song, we will be using A Classic Education by NightOwl from
+# MedleyDB (Creative Commons BY-NC-SA 4.0). This is also located in
+# `MUSDB18-HQ <https://zenodo.org/record/3338373>`__ dataset within
+# the ``train`` sources.
+# 
+# In order to test with a different song, the variable names and urls
+# below can be changed alongside with the parameters to test the song
+# separator in different ways.
+#
+
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+
+# We download the audio file from our storage. Feel free to download another file and use audio from a specific path
+SAMPLE_SONG = download_asset("tutorial-assets/hdemucs_mix.wav")
+waveform, sample_rate = torchaudio.load(SAMPLE_SONG) # replace SAMPLE_SONG with desired path for different song
+store = waveform
+
+# parameters
+segment: int = 10
+overlap = 0.1
+
+print(f"Separating track")
+
+ref = waveform.mean(0)
+waveform = (waveform - ref.mean()) / ref.std() # normalization
+
+sources = apply_model(
+    model,
+    waveform[None],
+    device=device,
+    segment=segment,
+    overlap=overlap,
+)[0]
+sources = sources * ref.std() + ref.mean()
+
+sources_list = model.sources
+sources = list(sources)
+
+audios = dict(zip(sources_list, sources))
+
+
+######################################################################
+# 5.1 Separate Track
+# ^^^^^^^^^^^^^^^^^^
+# 
+# The default set of pretrained weights that has been loaded has 4 sources
+# that it is separated into: drums, bass, other, and vocals in that order.
+# They have been stored into the dict “audios” and therefore can be
+# accessed there. For the four sources, there is a separate cell for each,
+# that will create the audio, the spectrogram graph, and also calculate
+# the SDR score. SDR is the signal-to-distortion
+# ratio, essentially a representation to the “quality” of an audio track.
+# 
+
+N_FFT = 4096
+N_HOP = 4
+stft = torchaudio.transforms.Spectrogram(
+    n_fft=N_FFT,
+    hop_length=N_HOP,
+    power=None,
+)
+
+
+######################################################################
+# 5.2 Audio Segmenting and Processing
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# 
+# Below is the processing steps and segmenting 5 seconds of the tracks in
+# order to feed into the spectrogram and to caclulate the respective SDR
+# scores.
+# 
+
+drums_original = download_asset("tutorial-assets/hdemucs_drums_segment.wav")
+bass_original = download_asset("tutorial-assets/hdemucs_bass_segment.wav")
+other_original = download_asset("tutorial-assets/hdemucs_other_segment.wav")
+vocals_original = download_asset("tutorial-assets/hdemucs_vocals_segment.wav")
+
+track = store
+mix_spec = track[:, 150*sample_rate: 155*sample_rate]
```

Comment:
constants `150` and `155` are repeated. Please define variables for them are reuse them. That way, users who are playing with the notebook on Colab have to change once for each variable.